frontend/src/libraries/tester.tsx

```diff
@@ -4,6 +4,17 @@ import renderer, {
 } from "react-test-renderer";
 import { LocalizeProvider } from "react-localize-redux";
 import LocalizeWrapper from "../components/utils/LocalizeWrapper";
+import { NetworkContext } from "../context/NetworkContext";
+
+const networkContext = {
+  currentNetwork: {
+    name: "testing",
+    explorerLink: "http://explorer/",
+    aliases: ["alias1"],
+    nearWalletProfilePrefix: "http://wallet/profile",
```

Comment:
```suggestion
    explorerLink: "http://explorer.near.org/",
    aliases: ["alias1"],
    nearWalletProfilePrefix: "http://wallet.near.org/profile",
```

Reply:
Apparently that doesn't affect anything as it's for test purposes only.
We'll fix that whenever we need actual addresses.